metricbeat/module/docker/container/_meta/fields.yml

```diff
@@ -40,3 +40,13 @@
           type: long
           description: >
              Size of the files that have been created or changed since creation.
+    - name: labels
+      type: dict
+      dict-type: keyword
+      description: >
+        Image labels.
+    - name: tags
+      type: list
+      dict-type: keyword
+      description: >
+        Image labels.
```

Comment:
No, my mistake. I updated the wrong file and then run `make update` :-( New version pushed.